apisix/schema_def.lua

```diff
@@ -472,11 +472,11 @@ local upstream_schema = {
         scheme = {
             default = "http",
             enum = {"grpc", "grpcs", "http", "https", "tcp", "tls", "udp",
-                "kafka"},
+                "kafka", "websocket"},
```

Comment:
Why was WebSocket added here? In APISIX, WebSocket is typically implemented via HTTP upgrades.